index.js

```diff
@@ -99,6 +114,16 @@ function throttle(options) {
             burst = options.overrides[attr].burst;
             rate = options.overrides[attr].rate;
         }
+        else if (options.overrides) {
```

Comment:
seems like this can just into into the `if` above and the `if` above can have two seconds? one for  instant IP dictionary lookup and the other for iterating the keys? If not, then at least a comment here saying this is where the 'cidr blocks' are handled by looking at all the overrides.